tests/manager/manager-executeBulkWrite_error-002.phpt

```diff
@@ -32,7 +32,7 @@ var_dump(iterator_to_array($cursor));
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-BulkWriteException: BulkWrite error :: E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 1 } :: E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 2 }
+BulkWriteException: Multiple write errors: "E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 1 }", "E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 2 }"
```

Comment:
@derickr: This is an example of the format change. libmongoc prepends "Multiple write errors" and then lists the messages wrapped in quotes and comma-delimited. The only other different I'm aware of is that the message will be capped at 504 bytes including the null terminator (since `bson_error_t` is statically allocated). I don't think that's a serious limitation, as the full messages will be available by inspecting the attached WriteResult object.